test/e2e/framework/profile_gatherer.go

```diff
@@ -107,8 +108,10 @@ func gatherProfileOfKind(profileBaseName, kind string) error {
 	}
 	defer outfile.Close()
 	cmd.Stdout = outfile
+	stderr := bytes.NewBuffer(nil)
+	cmd.Stderr = stderr
```

Comment:
This is cleaner IMO:

```
cmd.Stderr = &errbuf
return fmt.Errorf("Failed to run 'go tool pprof': %v, stderr: %v", err, errbuf.String())
```

Reply:
Sorry, the `&` there is not required. But the 2nd line still holds.

Reply:
done